src/logic/__tests__/startAccountMigration.test.js

```diff
@@ -0,0 +1,107 @@
+import {Observable} from 'rxjs';
+
+import startAccountMigration from '../startAccountMigration';
+import {
+  accountMigrationComplete,
+  accountMigrationError,
+  accountMigrationUndoPeriodExpired,
+} from '../../actions/user';
+import {migrateAccount} from '../../clients/firebase';
+import {getCurrentAccountMigration} from '../../selectors';
+import {bugsnagClient} from '../../util/bugsnag';
+
+import {
+  credentialFactory,
+  firebaseErrorFactory,
+  userFactory,
+} from '@factories/clients/firebase';
+
+import {
+  githubRepositoryFactory,
+} from '@factories/clients/github';
+
+
+jest.mock('../../actions/user.js');
+jest.mock('../../clients/firebase.js');
+jest.mock('../../selectors');
+jest.mock('../../util/bugsnag.js');
+
+const getState = jest.fn();
+const dispatch = jest.fn();
+const done = jest.fn();
```

Comment:
It doesn’t _really_ matter, but I think it’s a little nicer to put stuff like this inside the `describe` block they are being used by, just to make clear what their scope is. Feel free to ignore this though, works fine either way.